client/package.json

```diff
@@ -7,7 +7,6 @@
     "@emotion/react": "^11.4.0",
     "@emotion/styled": "^11.3.0",
     "@fullstory/browser": "^1.4.9",
-    "@stackoverflow/stacks-icons": "2.20.0",
     "@styled-icons/boxicons-logos": "^10.23.0",
     "@styled-icons/boxicons-regular": "^10.34.0",
```

Comment:
I see no reference to this in the codebase. Safe to remove?

Reply:
logos is used in footer, regular is used in editbutton

Reply:
we should probably get all of the icon packages from one source, but I don't know which source to use. Right now its 3/4 different one with no clear preferred way.

Reply:
we should use `react-icons/bi`, I would imagine.